### PR TITLE
Add new panel to figure with N contributors per month

### PR DESCRIPTION
**Pull request by adrn**
This addresses #150. The figure now looks like this:

commits

### CODE REVIEW

**Comment by adrn** [comment]
I actually think we could remove the middle panel and replace with the new right panel. What do others think?

**Review by crawfordsm — APPROVED** [review]

**Review by crawfordsm — CHANGES_REQUESTED** [review]
Actually, please remember to update the caption and the text relative to this figure.

**Comment by crawfordsm** [comment]
I'm okay with both -- commits and committers are different and I think they both provide some relevant information to the reader. 

It probably isn't easy to make, but I would be curious to see a second line of first time committers per month added to the last panel.

**Comment by bsipocz** [comment]
> It probably isn't easy to make, but I would be curious to see a second line of first time committers per month added to the last panel.

I suspect that line will be super noisy, around 4-5 in Feb-March, and much smaller during the summer. 

Also another measure could be along with the number of commits is either the number of PRs, or the cumulative lines of code (the latter I think the best of all three, number of commits varies much more than the number of lines for individual committers).

File: figures/commits_figure.py
Comment (hamogu) [inline figures/commits_figure.py:105]: clever! I learned something about how to use python here!

**Review by hamogu — COMMENTED** [review]
I'm not opposed to this change in general, but I'm opposed to it at this stage. I don't remember the referee asking for it (correct me if I'm wrong, it's been a few weeks since I reviewed the report) and I generally think of the content as mostly frozen after the paper is submitted. That has both practical reasons (a referee could trigger a new round of revision by asking for revisions / added discussion on the new content, while the existing figure already passed review) and philosophical reasons (we put the paper on astro-ph and people have cited it already. Once it's published, the ADS will merge that is the citations to the published paper and nobody will distinguish which authors originally cited arxiv v1 vs. arxiv v2 vs AJ). Obviously, we need to address the referee's comments and we need to correct any other errors we find. However, in this case, the existing figure is correct, this PR merely adds some additional information that may or may not be useful.

**Comment by bsipocz** [comment]
@hamogu - I disagree here, the referee specifically asked for providing more info about why we say this is still a healthy project. While I haven't looked into the link they suggested, my naive ideas about the health of a software project involves having a stable contributor group, and the new figure shows that to me. We still have a ~20 contributors/per month, so it's not the toy project of a very few individuals.

**Comment by hamogu** [comment]
Ok, that convinces me. Let's merge this!On Feb 22, 2018 7:43 AM, Brigitta Sipocz <notifications@github.com> wrote:@hamogu - I disagree here, the referee specifically asked for providing more info about why we say this is still a healthy project. While I haven't looked into the link they suggested, my naive ideas about the health of a software project involves having a stable contributor group, and the new figure shows that to me. We still have a ~20 contributors/per month, so it's not the toy project of a very few individuals.

**Comment by adrn** [comment]
@hamogu I think you agree now, but in case you need more convincing, I think the referee did explicitly ask for this panel! (see #150)

**Review by hamogu — APPROVED** [review]

**Review by crawfordsm — APPROVED** [review]

**Pull request merged**
